src/asymmetric_crypto_ctx.cpp

```diff
@@ -372,7 +372,9 @@ namespace mococrw {
                                  messageDigest.size());
             }
             catch (const OpenSSLException &e) {
-                throw MoCOCrWException(e.what());
+                throw MoCOCrWException((boost::format{"Signature validation failed. "
+                                                     "OpenSSL info: %1%"}
```

Comment:
Increase indentation by one space to line up below the string in the line before.